py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -76,12 +126,49 @@ def addmm_replacement(
     )
 
 
-@register_decomposition(torch.ops.aten.reciprocal.default, registry=DECOMPOSITIONS)  # type: ignore[misc]
+@register_torch_trt_decomposition(
+    torch.ops.aten.reciprocal.default, registry=TORCH_TRT_DECOMPOSITIONS
+)
 def reciprocal_replacement(
     input_: torch.Tensor,
 ) -> torch.Tensor:
     return torch.div(1, input_)
 
 
-def get_decompositions() -> Dict[OpOverload, Callable[..., Any]]:
-    return DECOMPOSITIONS
+@register_decomposition(torch.ops.aten.var_mean, registry=TORCH_TRT_DECOMPOSITIONS)
```

Comment:
Remove this registration, since it causes duplicate registration errors because `var_mean` with no overload encapsulates _all_ `var_mean` overloads.